docs/components/DrawerDocs.js

```diff
@@ -0,0 +1,159 @@
+const React = require('react');
+
+const { Button, Drawer } = require('mx-react-components');
+
+const Markdown = require('components/Markdown');
+
+const DrawerDocs = React.createClass({
+
+  getInitialState () {
+    return {
+      demoDrawerOpen: false,
+      currentPage: 3,
+      totalPages: 8
+    };
+  },
+
+  _handleDemoButtonClick () {
+    this.setState({
+      demoDrawerOpen: true
+    });
+  },
+
+  _handleDrawerClose () {
+    this.setState({
+      demoDrawerOpen: false
+    });
+  },
+
+  _handlePreviousClick () {
+    if (this.state.currentPage > 1) {
+      this.setState({
+        currentPage: this.state.currentPage - 1
+      });
+    }
+  },
+
+  _handleNextClick () {
+    if (this.state.currentPage < this.state.totalPages) {
+      this.setState({
+        currentPage: this.state.currentPage + 1
+      });
+    }
+  },
+
+  _renderDrawer () {
+    const styles = this.styles();
+
+    return (
+      <Drawer
+        breakPoints={{ large: 1200, medium: 1100, small: 1320 }}
```

Comment:
@redperk I looked at the code in the `Drawer` component and you're right, the small breakpoint is not used. I think you should remove it from this example.
```javascript
breakPoints={{ large: 1200, medium: 1100 }}
```